python/ray/serve/_private/replica.py

```diff
@@ -196,14 +196,17 @@ async def initialize_replica():
                     controller_handle,
                     app_name,
                 )
-                self._init_finish_event.set()
+                self._init_finish = True
```

Comment:
Doesn't look like this is used anywhere - should this be `_replica_init_finish`?